wasm/package-lock.json

```diff
@@ -0,0 +1,313 @@
+{
```

Comment:
Please add this file to `.gitignore`

Reply:
disagree, package-locks are supposed to be committed, is there a reason you want this ignored?

Reply:
In that scenario CI is correct to fail, as it's catching a developer error, where if the package.json is changed, the lock file needs updating.

Reply:
Sounds good to me. Let's keep the file.